contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -4,6 +4,7 @@ enable_cache: true
 distro: "debian"
 suites:
 - "jessie"
+- "stretch"
```

Comment:
Nit: Once we are past v2.0.7 and all is stable; we want to start removing jessie things unless they are for legacy sigs